README.md

```diff
@@ -158,6 +167,15 @@ If a proposed type matches zero atoms, the RJMCMC move is rejected.
 
 Currently, the acceptance criteria does not include the full Metropolis-Hastings acceptance criteria that would include the reverse probability.  This needs to be added in.
 
+### Elemental sampler
+
+Command line example: `smarty --samplertype elemental --element=8 --basetypes=examples/AlkEtOH/atomtypes/basetypes.smarts --initialtypes=examples/AlkEtOH/atomtypes/basetypes.smarts --decorators=examples/AlkEtOH/atomtypes/new-decorators.smarts  --molecules=examples/AlkEtOH/molecules/test_filt1_tripos.mol2 --reference=examples/AlkEtOH/molecules/test_filt1_ff.mol2 --iterations 100 --temperature=0`
+
+The elemental sampler has the same principle of the original sampler. However, the sampler will occurs for only one element (such as Oxygen, Carbon, Hydrogen, etc), which needs to be specified on the command line.
+
+The element number needs to be specified (--element=8 for Oxygen).
```

Comment:
"The element number needs to be specified by atomic number (--element=8 for oxygen)."